### PR TITLE
Clarify #3335

### PR DESCRIPTION
**Pull request by christiankral**
I added an additional comment to better understand where the import problem (conflict) came from

### CODE REVIEW

**Comment by dietmarw** [comment]
What are those commented imports for anyway? Whenever I see commented code it looks like unfinished WIP models which probably should not be inside the MSL (ideally)

**Comment by beutlich** [comment]
> 
> 
> What are those commented imports for anyway? Whenever I see commented code it looks like unfinished WIP models which probably should not be inside the MSL (ideally)

Yes, may be appropriate to remove this source code comment at all.

**Comment by christiankral** [comment]
> > What are those commented imports for anyway? Whenever I see commented code it looks like unfinished WIP models which probably should not be inside the MSL (ideally)
> 
> Yes, may be appropriate to remove this source code comment at all.

In this particular case I would keep the commented code, as the original code was re-written due to a OpenModelica issue. Further below in this model:

```modelica
  // c = sqrt(2)/m*sum({u[k]*exp(j*phi[k]) for k in 1:m});
  // Alternative implementation due to https://trac.openmodelica.org/OpenModelica/ticket/4496
  c.re = sqrt(2)/m*sum(u[k].re*cos(phi[k])-u[k].im*sin(phi[k]) for k in 1:m);
  c.im = sqrt(2)/m*sum(u[k].re*sin(phi[k])+u[k].im*cos(phi[k]) for k in 1:m);
```

I guess one day we will switch back to the original code; so the comment is the reminder on what the original code was.

File: Modelica/Electrical/QuasiStatic/Polyphase/Blocks/ToSpacePhasor.mo [hunk 1]
```diff
@@ -4,6 +4,8 @@ block ToSpacePhasor "Conversion: m phase -> space phasor"
   // import Modelica.ComplexMath.j;
   // import Modelica.ComplexMath.exp;
   // import Modelica.ComplexMath.sum;
```
Comment (dietmarw) [inline Modelica/Electrical/QuasiStatic/Polyphase/Blocks/ToSpacePhasor.mo]: But these can be removed, right?

Comment (beutlich) [reply Modelica/Electrical/QuasiStatic/Polyphase/Blocks/ToSpacePhasor.mo]: No, as they belong to the commented eq. `c = sqrt(2)/m*sum({u[k]*exp(j*phi[k]) for k in 1:m});`.

Comment (dietmarw) [reply Modelica/Electrical/QuasiStatic/Polyphase/Blocks/ToSpacePhasor.mo]: ah no I see now it's just for the particular equation. Never mind.

**Pull request merged**